curp/src/server/curp_node.rs

```diff
@@ -225,7 +233,7 @@ impl<C: 'static + Command> CurpNode<C> {
         curp_cfg: Arc<CurpConfig>,
         tx_filter: Option<Box<dyn TxFilter>>,
     ) -> Result<Self, CurpError> {
-        let (sync_tx, sync_rx) = mpsc::unbounded_channel();
+        let (sync_tx, sync_rx) = flume::unbounded();
```

Comment:
what's the difference of these two unbounded channels?

Reply:
Fixed